ui/cap-react/src/components/drafts/form/themes/grommet/templates/FieldTemplate.js

```diff
@@ -62,7 +78,12 @@ let FieldTemplate = function(props) {
         <span>
           <span
             style={{ color: "#000" }}
-            dangerouslySetInnerHTML={{ __html: label }}
+            dangerouslySetInnerHTML={{
+              __html:
+                uiSchema["ui:options"] && uiSchema["ui:options"].titleIsMarkdown
+                  ? sanitizedInput(markedInput(label))
+                  : sanitizedInput(label)
```

Comment:
Let's pass `innerHtml` only when `*isMarkdown`, and in other cases just pass normally 
```
{ 
         uiSchema["ui:options"] && uiSchema["ui:options"].titleIsMarkdown ?
         <span style={{ color: "#000" }}
             dangerouslySetInnerHTML={{  __html: sanitizedInput(markedInput(label)) }}
         /> :
         <span style={{ color: "#000" }}> {label}</span>
}
```